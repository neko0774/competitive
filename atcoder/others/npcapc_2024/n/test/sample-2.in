4 10
520471651 866160932 848899741 650346545
756377215 402412491 920748640 255249004
371442152 93295238 350011159 679848583
528399020 957465601 22001245 407745834
363922864 418156995 324388560 611306817
671914474 323815171 638034305 796072406
765823638 254662378 175686978 123364350
786531344 396717902 774813609 895184869
