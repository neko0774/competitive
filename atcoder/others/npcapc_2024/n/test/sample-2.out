890467395
623743197
432365684
555543126
145580696
550546744
959254454
836036617
945090197
106843161
866547396
